tests/compas/geometry/test_circle.py

```diff
@@ -0,0 +1,18 @@
+from compas.geometry import Circle
+from compas.geometry import Plane  # noqa: F401
+from compas.geometry import Point  # noqa: F401
+from compas.geometry import Vector  # noqa: F401
```

Comment:
Why are these imports here if they are not used?

Reply:
They are used in the `eval(repr())` construct.   I put the comments in to make my IDE not complain, but it may be that they aren't necessary for pylint... I didn't check.

Reply:
aha! good! then I guess pylint also needs the `noqa`. I would add a comment about why these are needed here then